packages/apps/workshop/stories/design-system/components/action-menu/action-menu-webcomponent.stories.js

```diff
@@ -2,14 +2,14 @@ import { action } from '@storybook/addon-actions';
 import { boolean } from '@storybook/addon-knobs';
 import { forModule } from 'storybook-addon-angularjs';
 
-import ActionMenu from '@ovh-ux/ui-kit.action-menu';
+import '@ovh-ux/ui-kit/dist/js/oui-action-menu';
```

Comment:
This is breaking the functioning of workshop application.
As you can see below with the `README.md` import, you can import without precising the path of the main file.

```suggestion
import '@ovh-ux/ui-kit.action-menu';
```